doc/source/installation.rst

```diff
@@ -0,0 +1,70 @@
+Installation
+============
+
+Depending on your experience, role, computing environment and expectations optimal hardware utilization, you have several ways to install the dependencies required to run PIConGPU.
```

Comment:
REQUIRED:
Please clarify what you mean with `role`?
The wording `expectations optimal hardware utilization` is not clear? (Should it be `..., expectations and optimal hardware utilization`?)